gtsam/geometry/Rot3.cpp

```diff
@@ -228,6 +228,12 @@ Rot3 Rot3::slerp(double t, const Rot3& other) const {
   return interpolate(*this, other, t);
 }
 
+/* ************************************************************************* */
+pair<Unit3, double> Rot3::axisAngle(const Rot3& other) const {
```

Comment:
Actually, there should be no other :-(
I think this should be a Rot3 static method. We can always do
axisAngle =Rot3::AxisAngle(R1.between(R2));
Which presents a problem , as AxisAngle already exists as a "named constructor". Maybe Rot3::ToAxisAngle ?

Reply:
Would having both make sense?

`Rot3::ToAxisAngle` as well as `R1.axisAngle(R2)`?

Reply:
No, I don't think so.